certified-connectors/Vineforce/apiDefinition.swagger.json

```diff
@@ -452,10 +490,15 @@
           {
             "name": "body",
             "in": "body",
-            "required": false,
+            "required": true,
```

Comment:
@nikunjvf This parameter has been changed from optional to required. existing flow may break due to this change. can you please confirm that no users will be affected by this change?

Reply:
@Amjed-Ayoub Yes it wont affect any users. Thanks.